.webpack/webpack.dev.js

```diff
@@ -11,6 +11,8 @@ const { merge } = require('webpack-merge');
 
 const base = require('./webpack.base');
 
+const projectRootDir = path.resolve(__dirname, '..');
```

Comment:
need to make sure we require `path` before we use it

Reply:
ups 🤭